### DISCUSSION TITLE
MPD crashes/won't start upon latest update

### DISCUSSION THREAD

**Discussion by khourycm** [rern/rAudio#1116] [Bug Reports]
updated to latest 1117 / 1123 version-- now MPD will not start upon bootup, or manual initiation (both on HTML interface and SSH/command line). Here is the error:

 MPD is not running 
mpd 0.22.3-2
# /srv/http/data/mpdconf/mpd.conf
audio_buffer_size   ""
auto_update         "yes"
bind_to_address     "0.0.0.0"
db_file             "/srv/http/data/mpd/mpd.db"
music_directory     "/mnt/MPD"
playlist_directory  "/srv/http/data/playlists"
state_file          "/var/lib/mpd/mpdstate"
user                "mpd"
input {
	plugin  "curl"
}
decoder {
	plugin   "ffmpeg"
	enabled  "yes"
}
audio_output {
	name    "Named pipe"
	type    "fifo"
	path    "/tmp/mpd.fifo"
	format  "44100:16:1"
}
resampler {
	plugin   "soxr"
	quality  "very high"
	threads  "1"
}
audio_output {
	name           "HiFiBerry DAC+ Pro"
	device         "hw:0,0"
	type           "alsa"
	auto_resample  "no"
	auto_format    "no"
	mixer_type     "software"
}

mpd.service- Music Player Daemon
     Loaded: loaded (/usr/lib/systemd/system/mpd.service; enabled; vendor preset: disabled)
    Drop-In: /etc/systemd/system/mpd.service.d
             └─override.conf
     Active: failed (Result: exit-code) since Sun 2022-11-27 18:41:23 EST; 41s ago
TriggeredBy: ● mpd.socket
       Docs: man:mpd(1)
             man:mpd.conf(5)
    Process: 1602 ExecStart=/usr/bin/mpd --systemd /srv/http/data/mpdconf/mpd.conf (code=exited, status=1/FAILURE)
   Main PID: 1602 (code=exited, status=1/FAILURE)

Nov 27 18:41:23 rAudio systemd[1]: Starting Music Player Daemon...
Nov 27 18:41:23 rAudio mpd[1602]: exception: Unknown option: --systemd
Nov 27 18:41:23 rAudio systemd[1]: mpd.service: Main process exited, code=exited, status=1/FAILURE
Nov 27 18:41:23 rAudio systemd[1]: mpd.service: Failed with result 'exit-code'.
Nov 27 18:41:23 rAudio systemd[1]: Failed to start Music Player Daemon.

**Comment by rern** [comment]
@khourycm,

Try: 
```sh
rm /srv/http/data/mpdconf/buffer.conf
/srv/http/bash/settings/player-conf.sh
```

**Comment by khourycm** [comment]
Thanks @rern-- tried it and got the following output (fail):

rAudio:~  #  systemctl status mpd.service^C
rAudio:~  #  rm /srv/http/data/mpdconf/buffer.conf
/srv/http/bash/settings/player-conf.sh
Job for mpd.service failed because the control process exited with error code.
See "systemctl status mpd.service" and "journalctl -xe" for details.
MPD error: Address family not supported by protocol
MPD error: Address family not supported by protocol
MPD error: Address family not supported by protocol
MPD error: Address family not supported by protocol
MPD error: Address family not supported by protocol
mpdnotrunning

**Comment by Vseason** [comment]
As a potentially related bug (rather than starting a new thread, since my audio output is at least outputting), as of 1123 I’ve noticed numerous new pops and clicks in playback. At first I thought it was either poor mastering or my imagination, but I’ve now confirmed audio that previously played without error as of 1010—not sure about 1014 or 1117, since I didn’t listen to those albums during that particular span—produces unrepeatable, yet persistently random, glitches. Something definitely changed!

You are likely already fixing it, so I’ll just send appreciation for all you do, and a bit of patience, because code is code and any minute jot or tittle can skew the result. :)

**Reply by Vseason** [reply]
“Unrepeatable, persistently random glitches” > Meaning that, when a noticeable pop or click occurs, I can go back and replay the same section, without the previously-noted pop or click.
